nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -4862,4 +4794,46 @@ mod tests {
             );
         }
     }
+
+    #[tokio::test]
+    async fn insert_target_query_sql_snapshot() {
+        use crate::db::raw_query_builder::expectorate_query_contents;
+
+        let query = InsertTargetQuery {
+            target_id: BlueprintUuid::nil(),
+            enabled: true,
+            time_made_target: Utc::now(),
+        };
+
+        expectorate_query_contents(
+            &query.to_query(),
+            "tests/output/insert_target_blueprint_query.sql",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn insert_target_query_is_valid() {
```

Comment:
similar nit: most of these tests are named `explain_{query}`; should we make this consistent? also, the expectorate test _also_ checks for the query's validity...

Reply:
Sure, renamed